### PR TITLE
Update some of dependencies and compile fix.

### PR DESCRIPTION
**Pull request by antygravity**
## Issue
- close ##675

## Overview (Required)
-  add jcenter() and google() repositories
-  update kotlin version and some of other dependencies
-  update supportLibrary which cause most of problems
-  fix compile errors

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: buildSrc/src/main/kotlin/Depends.kt [hunk 1]
```diff
@@ -94,6 +94,7 @@ object Depends {
     object Groupie {
         val core = "com.xwray:groupie:${Versions.groupie}"
         val binding = "com.xwray:groupie-databinding:${Versions.groupie}"
+        val kotlinExtensions = "com.xwray:groupie-kotlin-android-extensions:${Versions.groupie}"
```
Comment (takahirom) [inline buildSrc/src/main/kotlin/Depends.kt]: Probably you don't use this dependency 👀

File: model/build.gradle.kts [hunk 2]
```diff
@@ -7,13 +8,21 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
...
-        mavenCentral()
+        google()
+        jcenter()
     }
     dependencies {
         classpath(kotlin("gradle-plugin", Versions.kotlin))
     }
 }
 
+allprojects {
```
Comment (takahirom) [inline model/build.gradle.kts]: I think `allprojects` is not needed. 📁 
What do you think?

```
repositories {
    google()
    jcenter()
}
```

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/staff/item/StaffItem.kt [hunk 3]
```diff
@@ -13,7 +13,7 @@ data class StaffItem(
 ) : BindableItem<ItemStaffBinding>(staff.name.hashCode().toLong()) {
 
     override fun createViewHolder(itemView: View): ViewHolder<ItemStaffBinding> {
-        return ViewHolder(DataBindingUtil.bind(itemView))
+        return ViewHolder(DataBindingUtil.bind(itemView)!!)
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/staff/item/StaffItem.kt]: Probably we can use HogeHogeBinding.inflate(xxx) method. 👀 
But If you don't have time, I think that is fine as it is.

File: app/build.gradle.kts [hunk 4]
```diff
@@ -219,7 +219,8 @@ dependencies {
 }
 
 repositories {
-    mavenCentral()
+    jcenter()
```
Comment (panpanini) [inline app/build.gradle.kts]: please make sure `google()` is before `jcenter()`

**Comment by takahirom** [comment]
Thank you for resolving problems !! 👍 That's cool 🆒

File: model/build.gradle.kts [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import org.gradle.api.JavaVersion.VERSION_1_7
```
Comment (panpanini) [inline model/build.gradle.kts]: it looks like this import is unused 👀

File: build.gradle [hunk 6]
```diff
@@ -0,0 +1 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
```
Comment (panpanini) [inline build.gradle]: 👀 is this needed?

**Comment by takahirom** [comment]
Probably this is occurred by Kotlin DSL
Please see this issue.
https://github.com/DroidKaigi/conference-app-2018/issues/677

**Pull request closed without merging**